common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -747,6 +737,21 @@
                     self.isMouseFocus = false;
                     self.rendered = true;
                     self.state.el.addClass('is-captions-rendered');
+
```

Comment:
It looks like `attr()` from jQuery is escape automatically?

"The final step of DOM manipulation in JavaScript often happens using JQuery. There are some JQuery functions such as $.text(), $.attr() and $.val() that expect plain text strings and take care of HTML-escaping its input for you."